frontend/src/App.js

```diff
@@ -1,3 +1,4 @@
+import React from "react";
```

Comment:
This shouldn't be necessary in REact 17+ 

https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#eslint